src/WordPress.php

```diff
@@ -245,7 +248,7 @@ public function create( $args, $assoc_args ) {
 
 		$this->site_data['site_fs_path']       = WEBROOT . $this->site_data['site_url'];
 		$this->cache_type                      = \EE\Utils\get_flag_value( $assoc_args, 'cache' );
-		$this->site_data['site_ssl']           = \EE\Utils\get_flag_value( $assoc_args, 'ssl', '' );
+		$this->site_data['site_ssl']           = \EE\Utils\get_flag_value( $assoc_args, 'ssl', 'le' );
```

Comment:
This will also always add value regardless of the flag being passed or not.

Something like this can be used:

```
		$this->site_data['site_ssl'] = '';
		if( isset( $assoc_args['ssl'] ) ) {
			$this->site_data['site_ssl'] = empty( $assoc_args['ssl'] ) || true === ($assoc_args['ssl']) ? 'le' : $assoc_args[ 'ssl' ];
		}
```

This is a very simple implementation that will check if it is set in assoc args, then only work on adding the default value.